build/gulp/tasks/docs.ts

```diff
@@ -66,7 +66,7 @@ task(
 // ----------------------------------------
 
 const componentsSrc = [`${paths.posix.src()}/components/*/[A-Z]*.tsx`]
-const behaviorSrc = [`${paths.posix.src()}/lib/accessibility/Behaviors/*/[A-Z]*.ts`]
+const behaviorSrc = [`${paths.posix.src()}/lib/accessibility/Behaviors/*/[A-Z a-z]*.ts`]
```

Comment:
space should be omitted between `A-Z` and `a-z`

Reply:
another question is whether we should still preserve upper case letters as starting ones - as far as I understand, with the rename refactoring there shouldn't be any behaviors starting with capital letter anymore?

Reply:
I think you're right. We don't need uppercase any more. Thank you!